diff -Naurp greensql-fw-1.2.2/conf/greensql.conf greensql-fw-1.2.2.oden/conf/greensql.conf
--- greensql-fw-1.2.2/conf/greensql.conf	2010-01-25 16:40:09.000000000 +0100
+++ greensql-fw-1.2.2.oden/conf/greensql.conf	2010-01-26 15:19:02.144211493 +0100
@@ -9,7 +9,7 @@ dbtype=mysql
 [logging]
 # logfile - this parameter specifies location of the log file.
 # By default this will point to /var/log/greensql.log file in linux.
-logfile = /var/log/greensql.log
+logfile = /var/log/greensql/greensql.log
 # loglevel - this parameter specifies level of logs to produce.
 # Bigger value yelds more debugging information.
 loglevel = 10
diff -Naurp greensql-fw-1.2.2/src/config.cpp greensql-fw-1.2.2.oden/src/config.cpp
--- greensql-fw-1.2.2/src/config.cpp	2010-01-25 16:40:09.000000000 +0100
+++ greensql-fw-1.2.2.oden/src/config.cpp	2010-01-26 15:19:32.721180247 +0100
@@ -71,7 +71,7 @@ void GreenSQLConfig::init()
     
     log_level = 3;
 
-    log_file = "/var/log/greensql.log";
+    log_file = "/var/log/greensql/greensql.log";
 }
 
 bool GreenSQLConfig::load(std::string & path)
